### DISCUSSION TITLE
Is it possible to extend to support some syntax from mkdocs?

### DISCUSSION THREAD

**Discussion by cryptobench** [markdoc/markdoc#413] [Q&A] [answered]
I am currently migrating away from mkdocs to markdoc with nextjs. While I could almost 1:1 migrate my markdown files, some things doesn't render correctly.

Here's an example of mkdocs syntax:

```
!!! info

This is some info text that should be displayed and rendered with some styles.
```

Is it possible to extend markdoc so that it supports this syntax? (!!! info)

**Comment by nkohari-stripe** [comment, marked as answer]
Markdoc doesn't support that syntax, and there's not currently a way to add support for it. The Markdoc equivalent would be to create a custom tag like this:

```
{% info %}
This is some info text that should be displayed and rendered with some styles.
{% /info %}
```

This tag would render a React component which would apply whatever styling you wanted to the text within the tag. The Markdoc documentation contains some [instructions on how to implement custom tags](https://markdoc.dev/docs/tags#create-a-custom-tag).

**Reply by cryptobench** [reply]
Thank you very much for the reply. I will stick to tags.

Have a greay day!
